tests/models/test_user.py

```diff
@@ -420,16 +420,22 @@ def test_merge(self, mock_get_mailchimp_api):
             ],
         }
 
+        # Fields in which the two users may differ
+        different_fields = [
+            'date_modified'
+        ]
+
         # from the explicit rules above, compile expected field/value pairs
         expected = {}
         expected.update(calculated_fields)
         for key in default_to_master_user_fields:
             expected[key] = getattr(self.user, key)
 
         # ensure all fields of the user object have an explicit expectation
+        # except for 'different_fields'
         assert_equal(
             set(expected.keys()),
-            set(self.user._fields),
+            set(self.user._fields).difference(different_fields),
```

Comment:
@brianjgeiger I thought about adding it to the calculated_field but felt it was a bit of a stretch. The main reason to add a different_fields list is for future expansion, in case we add more fields to the user that allows different value for the merged users. But I guess the possibility of adding another such field is pretty low. In hindsight it's reasonable to add to the calculated_fields rather than excluding the different_fields, which makes it a bit less readable. I'll go ahead and do what you suggested.